src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -91,6 +95,9 @@ internal SparkContext(JvmObjectReference jvmObject)
         {
             _jvmObject = jvmObject;
             _conf = new SparkConf((JvmObjectReference)_jvmObject.Invoke("getConf"));
+            _version = new Version((string)_jvmObject.Jvm.CallStaticJavaMethod(
+                "org.apache.spark.deploy.dotnet.DotnetRunner",
+                "SPARK_VERSION"));
```

Comment:
I don't think this really needs to be cached.

Reply:
gotcha.